### PR TITLE
Testcase for registry workload when backed by OCS respin of ceph pods

### PR DESCRIPTION
**Pull request by Akarsha-rai**
Signed-off-by: Akarsha <akrai@redhat.com>

### CODE REVIEW

File: tests/e2e/registry/test_registry_pod_respin.py [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+    """
+
+    @pytest.fixture(autouse=True)
+    def init_sanity(self):
+        """
+        Initialize Sanity instance
+
+        """
+        self.sanity_helpers = Sanity()
+
```
Comment (OdedViner) [inline tests/e2e/registry/test_registry_pod_respin.py:35]: you can add (line 35)
self.image_path = None
If you reach the tear down stage before you reach line number 84 there will be an error.

Comment (Akarsha-rai) [reply tests/e2e/registry/test_registry_pod_respin.py:35]: agreed, added at line 42

**Comment by vasukulkarni** [comment]
can you run the tier1 please? Thanks

**Comment by Akarsha-rai** [comment]
> can you run the tier1 please? Thanks

Triggered tier1 jobs: https://ocs4-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/job/qe-trigger-test-pr/2512/

**Comment by Akarsha-rai** [comment]
> PR validation
> 
> Cluster Name:
> Cluster Configuration: conf/ocsci/production-aws-ipi.yaml
> PR Test Suite: tier1
> PR Test Path: tests/
> Additional Test Params:
> OCP VERSION: 4.5
> OCS VERSION: 4.5
> 
> [Job UNSTABLE](https://ocs4-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/job/qe-trigger-test-pr/2512//testReport) (some or all tests failed).

This is tier1 job, following test failed which is not caused by this particular PR.

Failed	tests/manage/mcg/test_bucket_creation.py::TestBucketCreation::test_bucket_creation[3-OC]
Failed	tests/manage/mcg/test_bucket_creation.py::TestBucketCreation::test_bucket_creation[3-CLI]
Failed	tests/manage/mcg/test_bucket_deletion.py::TestBucketDeletion::test_bucket_delete[3-CLI]
Failed	tests/manage/mcg/test_bucket_deletion.py::TestBucketDeletion::test_bucket_delete[3-OC]
Failed	tests/manage/mcg/test_bucket_deletion.py::TestBucketDeletion::test_bucket_delete_with_objects[CLI]
Failed	tests/manage/mcg/test_bucket_deletion.py::TestBucketDeletion::test_bucket_delete_with_objects[OC]
Failed	tests/manage/mcg/test_bucket_policy.py::TestS3BucketPolicy::test_basic_bucket_policy_operations
Failed	tests/manage/mcg/test_bucket_policy.py::TestS3BucketPolicy::test_bucket_policy_actions
Failed	tests/manage/mcg/test_bucket_policy.py::TestS3BucketPolicy::test_object_actions
Failed	tests/manage/mcg/test_multipart_upload.py::TestS3MultipartUpload::test_multipart_upload_operations
Failed	tests/manage/z_cluster/test_add_mds_to_cluster.py::TestCephFilesystemCreation::test_cephfilesystem_creation
Failed	tests/manage/z_cluster/test_must_gather.py::TestMustGather::test_must_gather

Registry tier1 test passed
Passed	tests/e2e/registry/test_pod_from_registry.py::TestRegistryImage::test_run_pod_local_image

**Pull request merged**